### ISSUE TITLE
Apply filter on ImageView or Bitmap to see preview?

### ISSUE THREAD

**Issue by khawarraza** [MasayukiSuda/GPUVideo-android#99]
Is there any way we can apply filter on ImageView or Bitmap to get a preview of the filter? Any workaround for this?

**Comment by mersanjay24** [comment]
@khawarraza 

Yes i was add the image view filter 
let me know if you want any help

**Comment by khawarraza** [comment]
@mersanjay24 yes, I have a bitmap and I want to apply filter on bitmap to render it on ImageView. Library only works with video files. Are you able to find a workaround for this?

**Comment by mersanjay24** [comment]
share me your email id will provide you zip file and intigration step

**Comment by Jeel7584** [comment]
@mersanjay24 Can we set this library filter on imageview?
